src/sage/schemes/curves/affine_curve.py

```diff
@@ -1829,20 +1889,31 @@ def braid_monodromy(self):
         each of this paths is the conjugated of a loop around one of the points
         in the discriminant of the projection of ``self``.
 
-        NOTE:
+        .. NOTE::
+
+            The projection over the `x` axis is used if there are no vertical asymptotes.
+            Otherwise, a linear change of variables is done to fall into the previous case.
 
-        The projection over the `x` axis is used if there are no vertical asymptotes.
-        Otherwise, a linear change of variables is done to fall into the previous case.
+        .. NOTE::
+
+            This functionality requires the ``sirocco`` package to be installed.
 
         EXAMPLES::
 
             sage: A.<x,y> = AffineSpace(QQ, 2)
             sage: C = A.curve((x^2-y^3)*(x+3*y-5))
-            sage: C.braid_monodromy()               # optional - sirocco
+            sage: C.braid_monodromy()                                   # needs sirocco
             [s1*s0*(s1*s2)^2*s0*s2^2*s0^-1*(s2^-1*s1^-1)^2*s0^-1*s1^-1,
              s1*s0*(s1*s2)^2*(s0*s2^-1*s1*s2*s1*s2^-1)^2*(s2^-1*s1^-1)^2*s0^-1*s1^-1,
              s1*s0*(s1*s2)^2*s2*s1^-1*s2^-1*s1^-1*s0^-1*s1^-1,
              s1*s0*s2*s0^-1*s2*s1^-1]
+            sage: T.<t> = QQ[]
+            sage: K.<a> = NumberField(t^3 + 2, 'a')
+            sage: A.<x, y> = AffineSpace(K, 2)
+            sage: Curve(y^2 + a * x).braid_monodromy()                  # needs sirocco
```

Comment:
```suggestion
            sage: Curve(y^2 + a * x).braid_monodromy()
```
This actually doesn't need `sirocco` since the check is done before such a call.